### PR TITLE
Remove backwards compatibility imports.

### LINKED ISSUE #59: 5.0: Remove things that were emitting a DeprecationWarning

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #59

### CODE REVIEW

**Comment by dataflake** [comment]
How do you usually handle reviews? I don't mind doing them unless you expect some other people to step forward.

**Comment by jamadden** [comment]
I am happy to listen to any interested reviewer!  Reviewers who I am familiar with and who have worked in the code before are especially appreciated, though. I doubt I would merge a non-trivial change without having a review from a previous committer or the project or other projects that use the project. (Unless I'm getting desperate for time. I usually try to send out a general plea for reviews before that point. I understand that everyone is busy and most people work on this in their spare time; plus its all too easy to suffer from burnout.) You certainly qualify on all those points!

On some projects where I know that some people have an interest and/or expertise, or have expressed a specific interest in a specific patch, or are considered the nominal "owner" because of their history with the project, I will try to ask them directly. If I miss someone, I'm happy to be corrected for the future.

**Comment by dataflake** [comment]
What I usually do is assign people straight away, people who have maintained the package or are well-versed with the code. I fear just putting up a PR won't necessarily entice people, especially on a "low-level plumbing" package like this.

File: src/zope/component/interfaces.py
Comment (dataflake) [inline src/zope/component/interfaces.py:23]: Do you use local linting? Just wondering about that line. I wouldn't mind adding a canonical linting configuration to `tox.ini`.

**Review by dataflake — APPROVED** [review]

File: src/zope/component/interfaces.py
Comment (jamadden) [reply src/zope/component/interfaces.py:23]: Yes, I've settled on pylint because it does actual code analysis and doesn't just complain about whitespace.

**Comment by jamadden** [comment]
Thank you!

**Pull request merged**
